hudi-common/src/main/resources/hbase-site.xml

```diff
@@ -1699,13 +1699,16 @@ possible configurations would overwhelm and obscure the important.
       Implementation of the status publication with a multicast message.
     </description>
   </property>
-  <property>
-    <name>hbase.status.listener.class</name>
-    <value>org.apache.hadoop.hbase.client.ClusterStatusListener$MulticastListener</value>
-    <description>
-      Implementation of the status listener with a multicast message.
-    </description>
-  </property>
+
+  <!-- Hbase related classes used in hudi bundle packages are in shaded pattern,
+  this config can cause ClassNotFoundException -->
+  <!--<property>-->
+  <!--  <name>hbase.status.listener.class</name>-->
+  <!--  <value>org.apache.hadoop.hbase.client.ClusterStatusListener$MulticastListener</value>-->
+  <!--  <description>-->
+  <!--    Implementation of the status listener with a multicast message.-->
+  <!--  </description>-->
+  <!--</property>-->
```

Comment:
Initialization of the listener in Hbase

https://github.com/apache/hbase/blob/c49f7f63fca144765bf7c2da41791769286dfccc/hbase-client/src/main/java/org/apache/hadoop/hbase/client/ConnectionImplementation.java#L302-L305

https://github.com/apache/hbase/blob/c49f7f63fca144765bf7c2da41791769286dfccc/hbase-client/src/main/java/org/apache/hadoop/hbase/client/ClusterStatusListener.java#L72-L73

Reply:
So, if I understand correctly, this property is being removed here because you want your override to take effect? If so, just remove it instead of commenting out.

Reply:
Ok, I will remove this conf.